packages/tables/frontend/sirius-components-tables/src/table/useTableTranslation.tsx

```diff
@@ -0,0 +1,121 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { MRT_Localization } from 'material-react-table';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+/**
+ * Hook providing translation for Material React Table.
+ *
+ * @author tgiraudet
+ */
+export const useTableTranslation = (): Partial<MRT_Localization> => {
```

Comment:
Just to be sure, did you check: https://www.material-react-table.com/docs/guides/localization?locale=fr